### DISCUSSION TITLE
How to create a video of Stellarium

### DISCUSSION THREAD

**Discussion by KenY713** [Stellarium/stellarium#2983] [General]
I'm trying to create a video of a time span in Stellarium. How would I do that?

I'm using Stellarium V.20.3 on a Windows 10 PC.

Thanks in advance for your help.

Ken Y
Glenside, PA

**Comment by gzotti** [comment]
Use the recording feature of the Win10 Game Bar.

**Comment by KenY713** [comment]
Thanks!
 
I’ll give it a try.
 
Ken Y
Glenside, PA

**Reply by KenY713** [reply]
How do I mark this as SOLVED?

**Reply by gzotti** [reply]
Probably the "General" category has no "question solved" feature.

**Comment by KenY713** [comment]
Thanks.
 
Ken Yavit
713 Chapel Road
Glenside, PA 19038
215-341-1113 (Voice/Text)
